core/Core.rdf

```diff
@@ -1516,7 +1794,7 @@ Services as products as well as software products will be considered in the next
 		<rdfs:label xml:lang="en-US">material product role</rdfs:label>
 		<skos:example xml:lang="en-US">a manufactured good has a material product role when a manufacturer offers it for sale; a drug product has a material product role when it is bought by a customer in a pharmacy; sea shells have a material product role when they are collected, packaged and offered for sale;</skos:example>
 		<iof-av:adaptedFrom>Oxford Languages, term by the name ‘product’; also Wikipedia, term by the name ‘goods’ (as used in economics) and in particular, tangible goods</iof-av:adaptedFrom>
-		<iof-av:firstOrderLogicAxiom>MaterialProductRole(x) → Role(x) ∧ ∃y,∃z((BuyingBusinessProcess(y) ∨ OfferingForSaleBusinessProcess(y) ∨SupplyingBusinessProcess(y)) ∧ MaterialEntity(z) participatesInAtSomeTime(z,y) ∧ roleOf(x,z))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>MaterialProductRole(x) → Role(x) ∧ ∃y,∃z((BuyingBusinessProcess(y) ∨ OfferingForSaleBusinessProcess(y) ∨ SupplyingBusinessProcess(y)) ∧ MaterialEntity(z) participatesInAtSomeTime(z,y) ∧ roleOf(x,z))</iof-av:firstOrderLogicAxiom>
```

Comment:
Subformula "MaterialEntity(z) participatesInAtSomeTime(z,y)" is missing a connective.